R/teal_data-get_code.R

```diff
@@ -11,7 +11,7 @@
 #' @param deparse (`logical`) flag specifying whether to return code as `character` (`deparse = TRUE`) or as
 #' `expression` (`deparse = FALSE`).
 #' @return
-#' Either a character vector or expression representing code used to create the requested data sets.
+#' Either string or expression representing code used to create the requested data sets.
```

Comment:
Now that we return an expression of length 1
```suggestion
#' Either string or an expression representing code used to create the requested data sets.
```